doc/get_started/how_to_build_a_java_app_from_scratch.md

```diff
@@ -1,231 +0,0 @@
-# How to write an Azure IoT Hub client application from scratch with Java on Windows
```

Comment:
I thought it had been removed previously.
I discussed this with @olivierbloch and he agreed that it should be removed - there is now a more in depth article about building a Java app from scratch here: https://azure.microsoft.com/en-gb/documentation/articles/iot-hub-java-java-getstarted/